src/impact-metrics/metric-client.ts

```diff
@@ -29,7 +31,25 @@ export class MetricsAPI extends EventEmitter {
     this.metricRegistry.gauge({ name, help, labelNames });
   }
 
-  incrementCounter(name: string, value?: number, featureName?: string): void {
+  private getFlagLabels(flagContext?: MetricFlagContext): MetricLabels {
+    let flagLabels: MetricLabels = {};
+    if (flagContext) {
+      for (const flag of flagContext.flagNames) {
+        const variant = this.variantResolver.forceGetVariant(flag, flagContext.context);
```

Comment:
we evaluate flag on your behalf but we don't emit impression events or count metrics for those